src/main/java/vg/civcraft/mc/civmodcore/locations/spatial/octrees/BaseOcTreeNode.java

```diff
@@ -0,0 +1,191 @@
+/*
+MIT License
```

Comment:
We don't do custom licensing in our plugins. We can't have everyone add their own license to whatever class they add. Feel free to give yourself credit as author, but please not along with a license copy paste in every major class

Reply:
Your base license is inadequate and would attribute royalties and ownership to some Jesse Berger. This is not acceptable since I do not license my work for others by transfering ownership, so yes, that copyright notice has to be there.